switchboard/builtins.py

```diff
@@ -53,9 +55,10 @@ def get_field_value(self, instance, field_name):
         return super(IPAddressConditionSet, self).get_field_value(instance,
                                                                   field_name)
 
-    def get_group_label(self):
+    def get_group_label(self):  # pragma: nocover
```

Comment:
In my opinion, if a function doesn't do anything worth asserting, then it ought not be unit tested. The original Gargoyle project used `pragma: nocover` in similar situations (e.g., `ModelDict` methods that were just wrappers around the underlying `dict` methods), so I'm extending that to the rest of the codebase in order to be consistent. That said, I'm not adamant about this, so I'm open to dropping the `pragma: nocover`s. I doubt it'll impact code coverage all that much, especially since I'll be happy with > 90%. On the flip side: I think using `pragma: nocover` actually makes the code coverage more useful and accurate--you're not losing percentage for statements that do nothing (and I'd include returning a static, constant string as "doing nothing").